Sources/CombustionBLE/DFUManager.swift

```diff
@@ -65,10 +65,9 @@ class DFUManager {
     func setDefaultDFUForType(dfuFile: URL?, dfuType: DFUDeviceType) {
         guard let dfuFile = dfuFile else { return }
         
-        do {
-            defaultFirmware[dfuType] = try DFUFirmware(urlToZipFile: dfuFile)
+        if let dfu = DFUFirmware(urlToZipFile: dfuFile) {
```

Comment:
Could you update the minimum version of the Nordic library in `Package.swift` to `4.15.3`?  Otherwise someone who uses this library may end up with a incompatible version of Nordic library.

Reply:
Sure @jjohnstz, thanks for the comment